pkgs/applications/misc/ptask/tw-version.patch

```diff
@@ -0,0 +1,11 @@
+--- a/src/tw.c	2017-02-21 22:56:19.289037608 +0100
++++ b/src/tw.c	2017-02-21 22:57:04.726814256 +0100
+@@ -116,7 +116,7 @@
+ 	    || !strcmp(ver, "2.3.0")
+ 	    || !strcmp(ver, "2.4.0")
+ 	    || !strcmp(ver, "2.4.1")
+-	    || !strcmp(ver, "2.5.0"))
++	    || !strcmp(ver, "2.5.1"))
```

Comment:
I haven't had the time yet to contribute it upstream. `ptask` has a `--force` switch to circumvent this switch, but this obviously degrades usability. It does work, though, as 2.5.1 is a maintenance release. I would leave the patch until it is fixed upstream. As `ptask` is meant to be a graphical interface, it does not make a lot of sense to force the user to go back to the terminal to make it work.